src/view/treeNodes/pullRequestNode.ts

```diff
@@ -607,6 +615,89 @@ export class PRNode extends TreeNode {
 		return null;
 	}
 
+	private async startDraft(_token: vscode.CancellationToken): Promise<void> {
+		await this._prManager.startReview(this.pullRequestModel);
+		this._onDidChangeCommentThreads.fire({
+			added: [],
+			changed: [],
+			removed: [],
+			inDraftMode: true
+		});
+	}
+
+	private updateCommentPendingState() {
+		const allComments = this._fileChanges
+			.reduce((all, change) =>
```

Comment:
Can we run a for loop / forEach to iterate through all file changes and their comments, instead of pushing them into a temporary array?